### PR TITLE
[Portuguese] Update tags to all terms

### PR DESCRIPTION
**Pull request by edsoncelio**
Signed-off-by: edsoncelio <edsoncelio2020@gmail.com>

### Describe your changes
Update tags to all terms.

### Related issue number or link (ex: `resolves #issue-number`)
based on https://github.com/cncf/glossary/pull/1449/files

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by brunoguidone — APPROVED** [review]
LGTM

File: content/pt-br/cloud-native-apps.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Aplicações Nativas em Nuvem 
 status: Completed
 category: conceito
-tags: ["aplicação", "", ""]
+tags: ["aplicação", "fundamentos", ""]
```
Comment (MrErlison) [inline content/pt-br/cloud-native-apps.md:5]: typo: fundamentoS

**Comment by MrErlison** [comment]
Hello @edsoncelio! Thank you very much for your contribution! 🎉

**Comment by MrErlison** [comment]
LGTM

**Review by MrErlison — APPROVED** [review]

**Pull request merged**